lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1318,7 +1319,10 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     unsigned unrollDFactor = 1;
 
     if ((destDepth % 8) == 0) {
```

Comment:
I think you can simplify this whole condition by only checking that `destDepth / CI->getGroup() % 8 == 0`. We can be sure that `destDepth % CI->getGroup() == 0` because of multiple verifiers higher in the stack.

Reply:
The test disabled DKKC8 opt:
Resnet50: 0.5457 (w group) vs. 0.5461 (w/o group)
vgg19      : 2.6721  (w group) vs.  2.6636 (w/o group)
squeezenet: 0.0714                 vs. 0.0738
zfnet512   : 0.4457                   vs. 0.4497

Reply:
Performance comparison looks good!